webapp/src/main/webapp/WEB-INF/conf/integ-ecms-social/ecms-social-configuration.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.exoplaform.org/xml/ns/kernel_1_1.xsd http://www.exoplaform.org/xml/ns/kernel_1_1.xsd"
+  xmlns="http://www.exoplaform.org/xml/ns/kernel_1_1.xsd">
+  <external-component-plugins>
+    <target-component>org.exoplatform.services.listener.ListenerService</target-component>  
```

Comment:
Do we really need all these ECMS listeners to configure in this add-on?

Reply:
Yes, we need to overwrite this configuration to **replace** FileUpdateActivityListener by our DocumentUpdateActivityListener. This will change the logic of comments. 
Warning: OnlyOffice add-on does the same thing, so we can't be sure which listener will be invoked in the case of two add-ons installed.

Reply:
According Vlad, we need replace the ECMS listener, thus need override a whole configuration file from ECMS.